ergoCubSN000/hardware/inertials/head-inertial.xml

```diff
@@ -43,7 +43,7 @@
             </group>
 
             <group name="SETTINGS">
-                <param name="acquisitionRate">      10      </param>
+                <param name="acquisitionRate">      20      </param>
```

Comment:
Just a curiosity, why the acquisition rate has been augmented to 20 (I suppose) ms?

Reply:
Yep, it is interpreted as a period in milliseconds.
@marcoaccame and I applied this change to avoid possible issues when reading the IMU from the new RFE master.
We may restore it to 10ms, if necessary, once the IMU works correctly.

Reply:
Uhm, I honestly avoid this. Let me revert this change. Local fiddling is always possible but changing the pristine original and intended value is risky.